recipe/magento2.php

```diff
@@ -124,12 +129,14 @@
 
 desc('Enables maintenance mode');
 task('magento:maintenance:enable', function () {
-    run("if [ -d $(echo {{current_path}}) ]; then {{bin/php}} {{current_path}}/bin/magento maintenance:enable; fi");
+    // do not use {{bin/magento}} because it would be in "release" but the maintenance mode must be set in "current"
```

Comment:
@peterjaap Notable change: when we force `current_path`, do not use `{{bin/magento}}`